charts/stable/wmbusmeters/values.yaml

```diff
@@ -0,0 +1,63 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: weetmuts/wmbusmeters
+  # -- image tag
+  tag: release-1.4.1-amd64
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+
+service:
+  main:
+    enabled: false
```

Comment:
If the app never would require a Service object, it might be better to move this to the hardcoded section.